tests/Kros.KORM.UnitTests/CommandGenerator/CommandGeneratorShould.cs

```diff
@@ -324,9 +376,9 @@ private IQuery<T> CreateQuery<T>()
             return query;
         }
 
-        private TableInfo GetFooTableInfo() => GetFooTableInfo(true);
+        private TableInfo GetFooTableInfo(bool isValueGeneratedOnInsertOrUpdate = false) => GetFooTableInfo(true, isValueGeneratedOnInsertOrUpdate);
 
-        private TableInfo GetFooTableInfo(bool withIdRow)
+        private TableInfo GetFooTableInfo(bool withIdRow, bool isValueGeneratedOnInsertOrUpdate)
```

Comment:
Toto je zbytočné robiť komplikovane cez `boolean`. Parameter nech rovno je typu `ValueGenerated`, default pôvodná hodnota a kde potrebuješ, si pošleš niečo iné.